saleor/saleor-checkout
# Adyen provider setup

QA instructions:

1. Create a new checkout and save it's `token`
2. Open it in the checkout deployment with `?checkoutToken=<token>` appended at the end
3. Complete checkout details (billing, shipping and payment method if required) and click pay
4. Adyen payment page should now be opened up
5. Complete the payment using a test card (see here: https://docs.adyen.com/development-resources/testing/test-card-numbers#visa)
6. The user should be redirected back to checkout with proper order confirmation view (the payment status will be unpaid for now)
